packages/mantine/src/components/prompt/Prompt.tsx

```diff
@@ -65,9 +66,9 @@ export const Prompt: PromptType = ({children, variant = 'info', ...otherProps})
         <Modal
             variant="prompt"
             padding={0}
-            classNames={{...classNames, ...otherProps?.classNames}}
+            classNames={{...classNames, ...classesProps}}
```

Comment:
[nitpick alert] I would go with `...classes` and `...other` but this is nitpicking indeed.